components/selection/selection-mixin.js

```diff
@@ -42,6 +42,11 @@ export const SelectionMixin = superclass => class extends RtlMixin(CollectionMix
 
 	static get properties() {
 		return {
+			/**
```

Comment:
I think we'd probably want to keep this hidden, since we would not want this showing up in `d2l-list`'s or `d2l-table-wrapper`'s documentation. 

Maybe name it something more semantic (maybe `selectionNoInputArrowKeyBehaviour` or something along those lines), in case the `keydown` handler ever needed to do something besides what it currently does.